jvm/src/main/scala/io/glutenproject/expression/UnaryOperatorTransformer.scala

```diff
@@ -94,7 +94,44 @@ class AbsTransformer(child: Expression, original: Expression)
     with ExpressionTransformer
     with Logging {
   override def doTransform(args: java.lang.Object): ExpressionNode = {
-    throw new UnsupportedOperationException("Not supported: Abs.")
+    val childNode = child.asInstanceOf[ExpressionTransformer].doTransform(args)
+    if (!childNode.isInstanceOf[ExpressionNode]) {
+      throw new UnsupportedOperationException(s"Not supported yet.")
+    }
+    val functionMap = args.asInstanceOf[java.util.HashMap[String, java.lang.Long]]
+    val functionId = ExpressionBuilder.newScalarFunction(functionMap,
+      ConverterUtils.makeFuncName(ConverterUtils.ABS, Seq(child.dataType), FunctionConfig.OPT))
+    val expressionNodes = Lists.newArrayList(childNode.asInstanceOf[ExpressionNode])
+    val nullable = original.nullable
+    val typeNode = original.dataType match {
+      case DoubleType =>
+        TypeBuilder.makeFP64(nullable)
+      case LongType =>
+        TypeBuilder.makeI64(nullable)
+      case IntegerType =>
+        TypeBuilder.makeI32(nullable)
+      case otherType =>
+        throw new UnsupportedOperationException(s"Type $otherType not supported in abs().")
+    }
```

Comment:
Please use `getTypeNode` function in ConverterUtils to get rid of this.

Reply:
Because ConverterUtils.getTypeNode is not fit to abs().

Reply:
It is used to convert Spark datatype into Substrait type node. I think above types are already covered by that function.

Reply:
Done